frontend/src/pages/manager/spaceHome/SpaceHomePage.tsx

```diff
@@ -37,15 +40,14 @@ import { track } from '../../../utils/googleAnalytics/track';
 import { goToTop } from '../../../utils/goToTop';
 import EarlyPage from '../../status/earlyPage/EarlyPage';
 import ExpiredPage from '../../status/expiredPage/ExpiredPage';
+import NoAccessPage from '../../status/noAccessPage/NoAccessPage';
 import * as S from './SpaceHomePage.styles';
 
 const SpaceHomePage = () => {
-  const { spaceCode } = useSpaceCodeFromPath();
-  const { spaceInfo } = useSpaceInfo(spaceCode ?? '');
-  const isEarlyTime =
-    spaceInfo?.openedAt && checkIsEarlyDate(spaceInfo.openedAt);
-  const isSpaceExpired = spaceInfo?.isExpired;
-  const spaceName = spaceInfo?.name ?? '';
+  const overlay = useOverlay();
+  const { showToast } = useToast();
+  const navigate = useNavigate();
```

Comment:
훅을 순서대로 그룹화해서 잘 정리해주셨군요🥹 한가지만 더 제안해본다면 `const { showToast } = useToast();`의 순서를 내려서 정리해볼 수 있겠어요.!

```suggestion
  const overlay = useOverlay();
  const navigate = useNavigate();
  
   const { showToast } = useToast();
   const { spaceCode } = useSpaceCodeFromPath();
   // ...
```